parts/chapters/subsections/12.3/WCONINJE.fodt

```diff
@@ -5235,7 +5235,7 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
     </table:table>
     <text:p text:style-name="P70">Table <text:sequence text:ref-name="refTable0" text:name="Table" text:formula="ooow:Table+1" style:num-format="1">12.78</text:sequence>: WCONINJE Keyword Description</text:p>
     <text:p text:style-name="P74"/>
-    <text:p text:style-name="P72">See also the GCONPROD the GCONINJ<text:span text:style-name="T54">E</text:span> keywords to define a group’s production and injection targets and constraints, and the WCONPROD keyword to define a production well’s targets and constraints. All the aforementioned keywords are described in the SCHEDULE section.</text:p>
+    <text:p text:style-name="P72">See also the GCONPROD the GCONINJE<text:span text:style-name="T54">E</text:span> keywords to define a group’s production and injection targets and constraints, and the WCONPROD keyword to define a production well’s targets and constraints. All the aforementioned keywords are described in the SCHEDULE section.</text:p>
```

Comment:
Please delete extra E

Reply:
Oops. Will change that. All these spans in the LibreOffice document can make it really hard to see what is going on properly sometimes.

Reply:
Thanks. I totally agree about spans.

Reply:
> I totally agree about spans.

@blattms Maybe we can write a script to remove all those span tags? I think they are only used for version control https://help.libreoffice.org/latest/en-US/text/shared/guide/redlining_versions.html, but we are already using Git for version control so we should not need them, right?